src/utils.mjs

```diff
@@ -0,0 +1,28 @@
+import process from 'node:process';
+
+/**
+ * Removes invalid filesystem characters from a given string.
+ * @param {string} str - The string to sanitize.
+ * @returns {string} - The sanitized string.
+ */
+const sanitize = (str) => str.replace(/[/\\?%*:|'<>]/g, '');
+
+/**
+ * Retrieves an environment variable from a GitHub Actions workflow or returns a default value.
+ * @param {string} name - The input name (used to construct the environment variable key).
+ * @param {string} [defaultVal] - The default value if the environment variable is not found.
+ * @returns {string} - The trimmed input value from the environment variable or default value.
+ * @throws {Error} - Throws an error if the input is required but not provided.
+ */
+const getInput = (name, defaultVal) => {
+  console.log(`INPUT_${name.replace(/ /g, '_').toUpperCase()}`);
+  const val = process.env[`INPUT_${name.replace(/ /g, '_').toUpperCase()}`] || '';
```

Comment:
```suggestion
  const inputName = `INPUT_${name.replace(/ /g, '_').toUpperCase()}`;
  console.log(inputName);
  const val = process.env[inputName] || '';
```

Or just remove the `console.log`, since it brings little value?
```suggestion
  const val = process.env[`INPUT_${name.replace(/ /g, '_').toUpperCase()}`] || '';
```